### ISSUE TITLE
bowpad color of find results selected item is not readable on Windows 7 in dark mode.

### ISSUE THREAD

**Issue by gmcode** [stefankueng/BowPad#123]
On Windows 7, in dark theme mode, if you do a search using bowpad's find/replace dialog, the results come back ok but the current selected item is not readable. The selected item always shows as a light blue background color and text that is in white or light blue background color. Ultimately that combination of colors for the selected item is unreadable. It might be better if in the dark theme, the text color was black for the selected item and the highlight colour could stay as it is which is a light blue on Windows 7.

**Comment by stefankueng** [comment]
screenshot please?

**Comment by gmcode** [comment]
You can see the white text with light grey background makes it pretty unreadable.

![bp](https://user-images.githubusercontent.com/10876766/102028657-2836a200-3e10-11eb-9508-13acf671da2a.png)

**Comment by stefankueng** [comment]
I'm sorry, but I have to close this issue: Win7 is no longer supported. Especially now that I've changed all bitmaps to pngs - which the Win7 ribbon does not load.

**Comment by gmcode** [comment]
Can I appeal for supporting Windows 7 just a while longer?

The main reason is that I am in the process of looking at finding/fixing several bugs that will benefit both Windows 7 and Windows 10 users, but there is not much incentive for me to do that if Bowpad is no longer supported on Windows 7 as that's what I'm using.

And to be fair to all Windows 7 users, dropping support for Windows 7 just to add .png icons that don't add any great value for anyone is probably not a good reason, where waiting a bit to get fixes that will help both sets of users is a better deal for everyone.

And maybe by then a better reason to stop supporting Windows 7 might exist and the Windows 7 version will be in a good shape at the time.

So can we hold off dropping Windows 7 support a little longer?
Thanks

**Comment by stefankueng** [comment]
I'm sorry, but no.
Win7 has been out of even extended support for almost a year now. That means using Win7 is not only dangerous for those who use it but also for all others because it's more and more likely that those machines become bots.
Apart from having more and more workaround to implement to still support Win7, I also don't want to encourage users to still use it by supporting it.

**Comment by gmcode** [comment]
Yes of course, because everyone is going to sign up to Windows 10 just so they can use Bowpad, and benefit from your encouragement, oh and get advertising id, more telemetry. Said nobody ever.

Alternatively, both Windows 10 and 7 users could benefit from me finding more bug instead and leaving the product in a usable from for Windows 7 users, if you waited a month. But whatever, I'm not surprised by your response. It's the typical negative knee jerk response and never a thanks for investigating.

**Comment by wmjordan** [comment]
I feel really very very sorry that an enhancement request has gotten the support to Windows 7 system terminated.
I did not expect this side-effect at that moment.

I still work with quite a few Windows 2008 machines--they are still supported by MS and BowPad has been performing as a great log file viewer on those machines. I had tried almost a dozen of text editors, and found that BowPad was the best for that job, years ago. The dropped support to Windows 7 system dramatically sentences the distinction of BowPad on Windows 2008 servers as well. That is really miserable.
